dev/com.ibm.ws.microprofile.faulttolerance_git_fat_tck/publish/servers/FaultToleranceTCKServer/server.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+    Copyright (c) 2018 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+   <featureManager>
+        <!--  Currently we set this version property in bootstrap.properties
+              we need to cover what we pull in in bnd.bnd: tested.features
+              as FeatureDependencyProcessor.java does not handle run time
+              properties so we can't fully infer this from cloned pom.xml's for now -->
+        <feature>mpFaultTolerance-${server.xml.feature.version}</feature>
```

Comment:
Instead of using this system property mechanism, I would recommend using the `RepeatTests` mechanism, which is documented here: https://github.com/OpenLiberty/open-liberty/wiki/FAT-tests#repeating-fat-tests-for-different-featurejee-levels

You can also see an example of this in build.example_fat in the FATSuite class

Reply:
(discussed on slack - requirement is slightly different)